app/res/values/themes.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <style name="CommonTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="CommonTheme" parent="Theme.MaterialComponents.Light.DarkActionBar.Bridge">

```

Comment:
Nice, `Bridge` is the reason why it's working on pre-lollipop right?

Reply:
No not really, Bridge theme is just a workaround for apps not comfortable with changing their theme to a MaterialComponent theme. Bridge theme inherits from the AppCompat Theme and adds any properties defined by the MaterialComponent theme in it. So it has nothing to do with device compatibility. The library support the 14+ devices by default.